fpa/jsonauditer/jsonauditer.go

```diff
@@ -0,0 +1,62 @@
+package jsonauditer
```

Comment:
This is a WIP.

The `jsonauditer` or `noopauditer` are implementations of an `Auditer` concept. This `Auditer` is a possible idea to have an event-log of events that happens in a Cid for an Fast API instance.

Reply:
Very cool idea, will be useful for sure. I'd love to display this data in the web ui.